javascript/core/objects/converting-to-and-from-json.md

```diff
@@ -7,6 +7,11 @@ levels:
 
 type: normal
 
+aspects:
+  - deep
```

Comment:
This is definitely `introduction` level, I would remove the `obscura` and `deep` labels.